nautobot/ipam/tests/test_views.py

```diff
@@ -263,6 +264,33 @@ def test_host_non_modifiable_once_set(self):
         self.assertEqual(200, response.status_code)
         self.assertIn("Host address cannot be changed once created", str(response.content))
 
+    def test_move_ip_addresses_between_namespaces(self):
+        ip_address = self._get_queryset().first()
+        new_namespace = Namespace.objects.create(name="Test Namespace")
+        # Assign model-level permission
+        obj_perm = ObjectPermission(name="Test permission", actions=["change"])
+        obj_perm.save()
+        obj_perm.users.add(self.user)
+        obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
```

Comment:
This can be replaced with: 
```suggestion
        self.add_permissions("ipam.change_ipaddress")
```